finals-2024-01-16/code/5142-simeon-milanov/src/app/services/student.service.ts

```diff
@@ -7,11 +7,11 @@ import { Student } from '../models/student';
 })
 export class StudentService {
 
-  private studentSubject = new BehaviorSubject<Student>({id: 1, name: 'John Doe'});
+  private studentSubject = new BehaviorSubject<Student>({id: 5142, name: 'Simeon Milanov'});
```

Comment:
+2 points for change in the footer service (no full points because it's not used correctly)